0.5/articles/webview.md

```diff
@@ -0,0 +1,325 @@
+---
+layout: default
+type: guide
+shortname: Articles
+title: "Using Polymer in a WebView"
+subtitle: How to setup Polymer in an Android WebView app
+
+article: true
+#draft: true
+description: How to build a mobile Polymer app that runs in a WebView.
+published: 2015-03-04
+#updated: 2014-10-20
+author: ebidel
+polymer_version: 0.5.5
+
+tags:
+- WebView
+- android
+- app
+- spa
+- cordova
+- hybrid
+---
+
+<style>
+#download-button {
+  background: #259b24;
+  color: #fff;
+  font-size: 18px;
+  fill: #fff;
+}
+#download-button:hover {
+  background: #0a7e07;
+}
+#download-button::shadow #ripple {
+  color: #fff;
+}
+</style>
+
+{% include authorship.html %}
+
+{% include toc.html %}
+
+## Introduction
+
+**Important** - This article assumes you are familiar with creating a WebView app on Android. If you're not familiar with WebView development, check out the excellent guides on the Android documentation, [Getting Started: WebView-based Applications for Web Developers](https://developer.chrome.com/multidevice/webview/gettingstarted) and [WebView for Android Overview](https://developer.chrome.com/multidevice/webview/overview).
+{: .alert .alert-info }
+
+Many developers ask us if {{site.project_title}} can be used inside a WebView.
+The answer is **of course**! Using {{site.project_title}} in a WebView is no different than creating a standalone web app that runs inside a WebView. However, in some cases it's not immediately obvious how to structure and app for WebView development. This guide walks you through starting a new Android WebView project and tweaking it to work with {{site.project_title}}. If you want to also target iOS, see the suggestions under [Writing one app across Web, Android 4.0+, and iOS](#oneapp).
+
+## Getting started
+
+Before you start, **develop a standalone web app first**. Leave out the fancy WebView stuff until the end. Why?
+
+- The Android Emulator is painfully slow to debug and test.
+- The emulator's [hardware acceleration](http://developer.android.com/tools/devices/emulator.html#acceleration) cannot be enabled for WebView apps.
+- It's much faster to iterate using your normal tools and workflow. 
+- The DevTools [remote USB debugging](https://developer.chrome.com/devtools/docs/remote-debugging) is second to none.
+
+Once sharp edges are ironed out, dive into WebView-fying and uploading it to Google Play.
```

Comment:
This is a really mixed metaphor :confused: Maybe "Once you've got the wrinkles ironed out ..." I would lean towards  "WebView-ifying" rather than "WebView-fying" but not strongly so.